packages/db/src/kysely-database.ts

```diff
@@ -0,0 +1,104 @@
+import type { Dialect, LogEvent, Logger as KyselyLogFunc } from 'kysely';
+import { CamelCasePlugin, Kysely, PostgresDialect } from 'kysely';
+import { Pool } from 'pg';
+
+import type { Logger } from '@intake24/common-backend';
+import type { DatabaseOptions, FoodsDB, SystemDB } from '@intake24/db';
+
+import type { DatabaseType } from './config';
+
+function databaseLogQuery(sql: string, logger: Logger, limit: number) {
+  if (limit > 0 && sql.length > limit) {
+    logger.debug(sql.substring(0, limit) + '...');
+  } else {
+    logger.debug(sql);
+  }
+}
+
+export class KyselyDatabases {
+  private readonly config;
+
+  private readonly env;
+
+  private readonly poolLogger: Logger;
+
+  private readonly dbLogger: Logger;
+
+  public foods!: Kysely<FoodsDB>;
+
+  public system!: Kysely<SystemDB>;
+
+  constructor({ environment, databaseConfig, logger }: DatabaseOptions) {
+    this.env = environment;
+    this.config = databaseConfig;
+    this.poolLogger = logger.child({ service: 'Kysely connection pool' });
+    this.dbLogger = logger.child({ service: 'Kysely DB' });
+  }
+
+  private configKyselyDialect(database: DatabaseType): Dialect {
+    // Postgres hard-coded for experimental Kysely code, in case we decide to keep using it
+    // return the correct dialect type based on the database config
+
+    return new PostgresDialect({
+      pool: new Pool({
+        database: this.config[this.env][database].database,
```

Comment:
Could you please add connection string option for kysely db? It's already in db config (`this.config[this.env][database].url`) for current setup and used in deployments.